src/Agent.Listener/Configuration/PromptManager.cs

```diff
@@ -0,0 +1,139 @@
+using Microsoft.VisualStudio.Services.Agent.Util;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.VisualStudio.Services.Agent.Listener.Configuration
+{
+    [ServiceLocator(Default = typeof(PromptManager))]
+    public interface IPromptManager: IAgentService
```

Comment:
I renamed this class from ConsoleWizard to PromptManager. This class is not the console wizard, ConfigurationManager is the console wizard - although technically it's only part of the wizard as it defers to other classes for service config. The point is, this class is not the console wizard, this class is just a prompt helper. Since all the convention in this repo is largely to use "___Manager" for helper classes, I renamed this class appropriately.